content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -155,6 +155,23 @@ This label has been deprecated. Please use `kubernetes.io/arch` instead.
 
 This label has been deprecated. Please use `kubernetes.io/os` instead.
 
+### kube-aggregator.kubernetes.io/automanaged {kube-aggregator-kubernetesio-automanaged}
+
+Example: `kube-aggregator.kubernetes.io/automanaged: "onstart"`
+
+Used on: APIService
+
+This label is attached to the APIService objects created by `kube-apiserver` to identify how they should be managed by `kube-apiserver`. This label should not be added to other APIService objects or modified on managed APIService objects.
+
+{{< note >}}
+Automanaged APIService objects are deleted by kube-apiserver when it has no built-in or custom resource API corresponding to the API group/version of the APIService.
+{{< /note >}}
+
+It has two possible values:
+* `onstart` indicates the APIService should be reconciled once at start.
+* `true` indicates the APIService should be reconciled continuously.
```

Comment:
I'd use a definition list; for example:
```suggestion

`onstart`
: the APIService should be reconciled when an API server starts up, but not otherwise.

`true`
: the API server should reconcile this APIService continuously.
```